open_bump.py

```diff
@@ -55,7 +55,9 @@ def get_kernel_size(image_name):
 
 def bumped(image_data):
     d = binascii.hexlify(image_data[-1024:])
-    return d.endswith(lg_magic) or d.startswith(lg_magic)
+    print(type(d))
+    print(lg_magic.encode("ASCII"), type(lg_magic.encode("ASCII")))
```

Comment:
Instead of this change, `lg_magic` can be changed to be a byte string on line 38 (`b"41...."`).

It's other uses will accept it